ops/internal/config/address.go

```diff
@@ -43,6 +45,30 @@ func (a ChecksummedAddress) String() string {
 	return common.Address(a).Hex()
 }
 
-func (a ChecksummedAddress) MarshalJSON() ([]byte, error) {
-	return []byte(fmt.Sprintf(`"%s"`, common.Address(a).Hex())), nil
+func (a *ChecksummedAddress) UnmarshalJSON(data []byte) error {
```

Comment:
Why do we need to change the serialization?

Reply:
We get errors when using Go's standard JSON unmarshaler because it doesn't know how to convert a string into the ChecksummedAddress type. Using this custom unmarshal means if incoming addresses (from state.json, etc) are not checksummed, we can still read them, then when we write them they will be checksummed

Reply:
Its the same logic as the custom UnmarshalTOML that already exists in this file. Since we are now reading from `addresses.json`, instead of only writing to it, it makes sense to have the custom UnmarshalJSON to me.

Reply:
Want to fail if read address is not checksummed

Reply:
Cleaned this up and used a helper method to encapsulate the code used by both UnmarshalToml and UnmarshalJSON